src/components/ToDoItem.vue

```diff
@@ -44,7 +45,7 @@
   <ToDoForm
     v-if="isFormEditable"
     :todo="todo"
-    :index="index"
+    :id="todo.id"
```

Comment:
Dont need to add an id prop cause you already passing the whole todo to the component